ait/gui/static/js/ait/util.js

```diff
@@ -36,5 +36,24 @@ function move (array, from, to) {
     array.splice(to, 0, array.splice(from, 1)[0])
 }
 
+/**
+ * Returns DN to EU value if it exists unless the `raw` parameter is true,
+ * otherwise return raw value.
+ */
+function getPacket (data, raw=false) {
```

Comment:
Not that the logic below is wrong, but couldn't we simplify a bit?

```
// Default to the raw value since it will always be avilable
let value = data['raw']

// If raw==false and a DN to EU value exists, grab that
if (!raw && data['dntoeu']) {
    value = data['dntoeu']
}

return value
```